server/global/Tokens.js

```diff
@@ -0,0 +1,68 @@
+import jwt from "jsonwebtoken";
+
+import { queryPromise } from "../db/database.js";
+
+/**
+ * Create an access token from a refresh token
+ * @param {string} refreshToken The refreshToken of the user
+ * @returns {string} The token
+ */
+function createAccessToken(refreshToken) {
+  const { ACCESS_TOKEN_KEY, REFRESH_TOKEN_KEY } = process.env;
```

Comment:
Should we check here that the values are not `undefined`?

Reply:
Even if the server cannot start without these keys ?

Reply:
Oh yeah that's correct. In the unlikely case when they are deleted at runtime, the server won't see these changes until restart, right?

Reply:
This is correct, but if there are `undefined` at this time, the function will throw an error, and the server will send a server error and log the error, that is the expected behavior